frontend/src/metabase/metabot/components/MetabotQueryBuilder/MetabotQueryBuilder.tsx

```diff
@@ -35,7 +36,7 @@ interface StateProps {
 type MetabotQueryBuilderProps = StateProps;
 
 const mapStateToProps = (state: State): StateProps => ({
-  loadingMessage: PLUGIN_SELECTORS.getLoadingMessage(state),
```

Comment:
Replacing all direct usages of `PLUGIN_SELECTORS.getLoadingMessage` and `PLUGIN_SELECTORS.getIsWhiteLabeling` with [separate functions](https://github.com/metabase/metabase/pull/32651/files#diff-d3c83822a3411d314510ea9296a46fede15c579fb3ddee3e817d20965a01ff5bR1) that are unit tested.